### PR TITLE
kubelet: remove the pleg health check from healthz

### PR DESCRIPTION
**Pull request by yujuhong**
This prevents kubelet from being killed when docker hangs.

Also, kubelet will report node not ready if PLEG hangs (`docker ps` + `docker inspect`).

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37865)

**Comment by yujuhong** [comment]
@k8s-bot kops aws e2e test this

**Comment by yujuhong** [comment]
@k8s-bot kops aws e2e test this

**Comment by yujuhong** [comment]
@dchen1107 ping!

**Comment by yujuhong** [comment]
@Random-Liu, could you help review the PR? Thanks!

File: pkg/kubelet/runtime.go [hunk 1]
```diff
@@ -81,6 +97,12 @@ func (s *runtimeState) runtimeErrors() []string {
 	if s.internalError != nil {
 		ret = append(ret, s.internalError.Error())
 	}
+	for _, hc := range s.healthChecks {
+		if ok, err := hc.fn(); !ok {
+			ret = append(ret, fmt.Sprintf("%s is not healthy: %s", hc.name, err))
```
Comment (Random-Liu) [inline pkg/kubelet/runtime.go]: `%v`?

**Review by Random-Liu — COMMENTED** [review]
LGTM with one nit

File: pkg/kubelet/runtime.go (hunk 1 above)
Comment (yujuhong) [inline pkg/kubelet/runtime.go]: fixed

**Comment by dchen1107** [comment]
LGTM except above nits from @Random-Liu

**Comment by yujuhong** [comment]
Addressed the nit.

**Comment by yujuhong** [comment]
@k8s-bot cvm gke e2e test this

**Comment by yujuhong** [comment]
Applied the lgtm label based on previous comments.

**Comment by ApsOps** [comment]
@Random-Liu Shouldn't this be back-ported or cherry-picked in v1.5? I see kubelet posting `Ready` status when docker runtime is hung.

**Pull request merged**